### PR TITLE
Typo in Javadoc

### PR DESCRIPTION
**Pull request by astubbs**

### CODE REVIEW

File: clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java [hunk 1]
```diff
@@ -213,7 +213,7 @@
  * <h4>Manual Offset Control</h4>
  *
  * Instead of relying on the consumer to periodically commit consumed offsets, users can also control when records
- * should be considered as consumed and hence commit their offsets. This is useful when the consumption of the
+ * should be considered as consumed and hence commit their offsets. This is useful when the consumption of the messages
```
Comment (enothereska) [inline clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java:216]: Should it be "...consumption...is coupled"? Currently it is "...consumption...are coupled".

Comment (astubbs) [reply clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java:216]: hmm, I don't /think/ so, but I'm not English major!

Comment (ijuma) [reply clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java:216]: @enothereska is right, if you make that additional change, I'll merge your PR. Thanks!

Comment (astubbs) [reply clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java:216]: @enothereska @ijuma done!

**Comment by ijuma** [comment]
Thanks for the PR, LGTM.

**Pull request closed without merging**